vcs/VTKPlots.py

```diff
@@ -14,6 +14,11 @@
 import vcsvtk
 
 
+def _makeEven(val):
+    if (val & 0x1):
+        val -= 1
```

Comment:
@doutriaux1 Feel free to change that. My though was to not go beyond the screen size. I think both += or -= would work as well.